torch/fx/graph.py

```diff
@@ -140,8 +141,11 @@ def _name(self, target: Target) -> str:
             if _is_magic(op):
                 op = op[2:-2]
         op = op.replace('.', '_')
-        op = op.replace('*', '')
+        # delete all characters that are illegal in a Python identifier
+        op = re.sub('[^0-9a-zA-Z_]', '', op)
```

Comment:
maybe instead of deleting - replace with `_`. E.g. if someone creates `foo/bar` as a name, `foo_bar` is better replacement than `foobar`. Maybe `re.sub('[^0-9a-zA-Z_]+', '_', op)` to replace a sequence of banned chars with one